msg/en_US/utils.msg

```diff
@@ -1276,7 +1276,7 @@ $set 58 MSGCAT_UTIL_SET_FLASHBACK
 14 Flashback cannot run as standalone mode.\n
 60 \
 flashback: Provides statements to flashback a specific transaction.\n\
-usage: %1$s flashback [OPTION] database-name class_name1 [class_name2 ...]\n\
+usage: %1$s flashback [OPTION] database-name owner_name.class_name1 [owner_name.class_name2 ...]\n\
```

Comment:
The owner name is not necessary. Don't we need to tell users that owner_name will be set to dba if not given?
```suggestion
usage: %1$s flashback [OPTION] database-name [owner_name.]class_name1 [[owner_name.]class_name2 ...]\n\
```

Reply:
You are right. This is the main concern for this modification. 
users can run it without specifying the owner name. 

But I think it's better to entice them to run by specifying the owner name to get the desired result.

Reply:
Ok, I understand. However, if omitting owner_name possibly makes a wrong result, I can't find the point to allow this. How about just preventing it?

Reply:
There seem to be two ways.
1. Proceed as it is now, but if the owner is not entered, **the manual** specifies that a table owned by dba will be searched.
2. If the owner is not specified, an error is emitted from the flashback.

I think 2 is better, and other utilities that take the class names as arguments should also be fixed like this.

but it is related to flashback spec, so I think we need to discuss it a bit more and decide.

Reply:
Yep, leave a comment here when it is decided, or have a discussion here by mentioning others.

Reply:
The decision was made according to your opinion.
We will force the user to specify the owner.

Reply:
The decision was made according to your opinion.
We will force the user to specify the owner.

Reply:
Thank you for letting me know!